pkg/querier/queryrange/query_range_test.go

```diff
@@ -18,14 +18,16 @@ import (
 )
 
 func TestRequest(t *testing.T) {
+	r := *parsedRequest
```

Comment:
possibly deserves a comment why you are copying?

Reply:
Done.